### PR TITLE
[fx] methods to get fx graph property.

### PR DESCRIPTION
**Pull request by YuliangLiu0306**

### CODE REVIEW

File: colossalai/fx/passes/utils.py [hunk 1]
```diff
@@ -25,3 +27,134 @@ def get_comm_size(prev_partition, next_partition):
...
+def is_leaf(graph: Graph, node: Node):
+    return node in get_leaf(graph)
+
+
+def get_top(graph: Graph):
+    """
+    Given a graph, return top nodes of this graph.
+
+    Note: This method will get the top nodes of given graph excluding ``placeholder`` nodes.
+    """
```
Comment (FrankLeeeee) [inline colossalai/fx/passes/utils.py:61]: Top nodes are quite confusing, will `get_placeholder` be better?

Comment (YuliangLiu0306) [reply colossalai/fx/passes/utils.py:61]: This method is used to get the root node after removing 'placeholder' and 'root' node from the origin fx graph.

Comment (FrankLeeeee) [reply colossalai/fx/passes/utils.py:61]: Ok, can you add this definition of top node in the docstring as it is not a common terminology?

Comment (YuliangLiu0306) [reply colossalai/fx/passes/utils.py:61]: Sure

**Review by FrankLeeeee — APPROVED** [review]

**Pull request merged**